README.md

```diff
@@ -1,3 +1,28 @@
 # Tax Tribunals Fees
 
 [![CircleCI](https://circleci.com/gh/ministryofjustice/tax-tribunals-fees.svg?style=svg&circle-token=53059f2bf1c3a736853b21bdb4ab3df9baf2dd2b)](https://circleci.com/gh/ministryofjustice/tax-tribunals-fees)
+
+Ruby on Rails Web application to enable payments for Tax Tribunal fees.
+
+## Setup & Run (locally)
+
+This app uses [Docker Compose](https://docs.docker.com/compose/) to run locally, which lets you spin up an app container and a database with minimal effort:
+
+```
+# Set up environment variables
+cp .env.example .env
```

Comment:
Worth pointing out that `DATABASE_URL` should be removed from `.env` and added to `.end.development` and `.env.test`, respectively:

`.env.test: DATABASE_URL=postgresql://postgres@localhost:5432/payments_alpha_test`
`.env.development: DATABASE_URL=postgresql://postgres@localhost:5432/payments_alpha_development`

for example.